apps/condo/pages/_app.tsx

```diff
@@ -222,18 +296,36 @@ const MenuItems: React.FC = () => {
                 )
             }
             <div>
-                {menuItemsData.map((menuItemData) => (
-                    <MenuItem
-                        id={menuItemData.id}
-                        key={`menu-item-${menuItemData.path}`}
-                        path={`/${menuItemData.path}`}
-                        icon={menuItemData.icon}
-                        label={menuItemData.label}
-                        disabled={disabled}
-                        isCollapsed={isCollapsed}
-                        toolTipDecorator={disabled ? wrapElementIntoNoOrganizationToolTip : null}
-                        excludePaths={menuItemData.excludePaths}
-                    />
+                {menuCategoriesData.map((category) => (
+                    <>
+                        {category.items.map((item) => (
+                            <MenuItem
+                                id={item.id}
+                                key={`menu-item-${item.path}`}
+                                path={`/${item.path}`}
+                                icon={item.icon}
+                                label={item.label}
+                                disabled={disabled}
+                                isCollapsed={isCollapsed}
+                                toolTipDecorator={disabled ? wrapElementIntoNoOrganizationToolTip : null}
+                                excludePaths={item.excludePaths}
+                            />
+                        ))}
+                        {get(contextsByCategories, category.key, []).map((ctx) => (
+                            <MenuItem
+                                id={`menu-item-app-${ctx.app.id}`}
+                                key={`menu-item-app-${ctx.app.id}`}
+                                path={`/miniapps/${ctx.app.id}`}
+                                icon={get(AllIcons, ctx.app.icon, null)}
```

Comment:
Icon may be missing. It will look strange with the sidebar collapsed. Can you specify a default icon instead of `null`?